apis/v1alpha2/nginxproxy_types.go

```diff
@@ -390,6 +390,11 @@ type KubernetesSpec struct {
 
 // Deployment is the configuration for the NGINX Deployment.
 type DeploymentSpec struct {
+	// Container defines container fields for the NGINX container.
+	//
+	// +optional
+	Container ContainerSpec `json:"container"`
+
```

Comment:
Looks like some unnecessary changes to the DeploymentSpec and DaemonSetSpec?

Reply:
Wonder why the linter would care now? It's been in this order for awhile with no issue.

Reply:
Not a huge deal if it's blocking you from using the old order.

Reply:
Ah, good point.